analysis/node-wrapper/install.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+var isWindows = require("is-windows");
+
+if (isWindows() === true && process.env.npm_package_name === "codecharta-analysis") {
```

Comment:
```suggestion

if (process.platform === "win32") {
```

This should only be executed in the analysis anyway, no?

And instead of indenting all the code, it's possible to use an early return:

```js
if (process.platform !== 'win32') {
  return;
}
// Other code
```

Reply:
> This should only be executed in the analysis anyway, no?

The name check is intended to verify if it is installed standalone and not as a dependency. But I am not sure if this works in that case.